File: sorting/selection-sort/C++/selection.cpp
#include <iostream>

int main() {
   int arr[]={9,6,5,7,2};
    int n = sizeof(arr)/sizeof(arr[0]);
    int temp;
    for(int i=0;i<n;i++){
        int min_index=i;
        for(int j=i+1;j<n;j++){
            if(arr[j]<arr[min_index]){
                min_index=j;
            }
        }
        temp = arr[min_index];
        arr[min_index]=arr[i];
        arr[i]=temp;
       
    }
    
    for(int i=0;i<n;i++){
        std::cout<<arr[i]<<" ";
    }
}

File: sorting/selection-sort/javascript/selection.js
let array = [5, 2, 3, 9, 1];
n = array.length;
for (let i = 0; i < n; i++) {
  let min_index = i;
  for (let j = i + 1; j < n; j++) {
    if (array[j] < array[min_index]) {
      min_index = j;
    }
  }
  let temp = array[min_index];
  array[min_index] = array[i];
  array[i] = temp;
}
for (let i = 0; i < n; i++) {
  console.log(array[i]);
}
